bundler/tool/bundler/dev_gems.rb.lock

```diff
@@ -35,8 +35,8 @@ PLATFORMS
   ruby
   universal-java-11
   universal-java-18
-  x64-mingw-ucrt
   x64-mingw32
+  x64-unknown
```

Comment:
why is this changing?

Reply:
I have no idea, I'll investigate in the history where this came from